scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -277,21 +277,17 @@ def run_build(self, args=[]):
     def run_cmake(self, args=""):
 
         if not self.options.disable_warnings_as_errors:
-            ldflags = "-Wl,--fatal-warnings"
-            cflags = "-Werror"
-            aflags = "-Werror -Wa,--fatal-warnings"
+            force_warnings_as_error = 1
```

Comment:
Why `force` in the name ?

Wouldn't `warnings_as_errors` suffice.

Reply:
Done